react-bootstrap-table/react-bootstrap-table.d.ts

```diff
@@ -520,11 +558,13 @@ declare module "react-bootstrap-table" {
 		options?: any;
 	}
 	
+	export type FilterType = 'TextFilter' | 'RegexFilter' | 'SelectFilter' | 'NumberFilter' | 'DateFilter' | 'CustomFilter';
```

Comment:
Ah thanks. I agree with your interpretation of the code. Also good idea on how to extend the type.